params/config.go

```diff
@@ -925,3 +925,22 @@ func NetworkIDByChainName(chain string) uint64 {
 		return config.ChainID.Uint64()
 	}
 }
+
+func Eth2DeployBlockNumber(c *ChainConfig) uint64 {
```

Comment:
return uint64 as a pointer and return nil if it does not exist for a specific chain config